docs/_developer_onboarding.md

```diff
@@ -117,41 +117,42 @@ You can also have access to the aars through the [release](https://github.com/ma
 
 #### Build the SDK locally
 
-Easiest way: run the script [../tools/sdk/build_rust_sdk.sh](../tools/sdk/build_rust_sdk.sh) and just answer the questions.
-
-Legacy way:
-
-If you need to locally build the sdk-android you can use
-the [build](https://github.com/matrix-org/matrix-rust-components-kotlin/blob/main/scripts/build.sh) script.
-
-For this please check the [prerequisites](https://github.com/matrix-org/matrix-rust-components-kotlin/blob/main/README.md#prerequisites) from the repo.
-
-Checkout both [matrix-rust-sdk](https://github.com/matrix-org/matrix-rust-sdk) and [matrix-rust-components-kotlin](https://github.com/matrix-org/matrix-rust-components-kotlin) repositories
-```shell
-git clone git@github.com:matrix-org/matrix-rust-sdk.git
-git clone git@github.com:matrix-org/matrix-rust-components-kotlin.git
-```
-
-Then you can launch the build script from the matrix-rust-components-kotlin repository with the following params:
-
-- `-p` Local path to the rust-sdk repository
-- `-o` Optional output path with the expected name of the aar file. By default the aar will be located in the corresponding build/outputs/aar directory.
-- `-r` Flag to build in release mode
-- `-m` Option to select the gradle module to build. Default is sdk.
-- `-t` Option to to select an android target to build against. Default will build for all targets.
-
-So for example to build the sdk against aarch64-linux-android target and copy the generated aar to Element X project:
-
-```shell
-./scripts/build.sh -p [YOUR MATRIX RUST SDK PATH] -t aarch64-linux-android -o [YOUR element-x-android PATH]/libraries/rustsdk/matrix-rust-sdk.aar
-```
+Prerequisites:
+* Install the Android NDK (Native Development Kit). To do this from within
+  Android Studio:
+    1. **Tools > SDK Manager**
+    2. Click the **SDK Tools** tab.
+    3. Select the **NDK (Side by side)** checkbox
+    4. Click **OK**.
+    5. Click **OK**.
+    6. When the installation is complete, click **Finish**.
+* Install `cargo-ndk`:
+  ```
+  cargo install cargo-ndk
+  ```
+* Install the Android Rust toolchain:
+  ```
+  rustup target add aarch64-linux-android
```

Comment:
You may want to add some other archs, since this is only for MacOS devices with ARM arch. The expected Android architectures are:

```
x86_64-linux-android
aarch64-linux-android
armv7-linux-androideabi
i686-linux-android
```

Reply:
well, the `build_rust_sdk.sh` script (which is what this section is all about) hardcodes `aarch64-linux-android`. That seems sensible to me: does anyone actually use any of those other archs for local development?

Sure you might need the extra archs if you're doing something exotic, but then you wouldn't be using `build_rust_sdk.sh`, and the instructions would belong somewhere else.